### PR TITLE
Release Godot 4.3 dev 1 and publish release notes

### PR DESCRIPTION
**Pull request by YuriSizov**
This ^

### CODE REVIEW

File: collections/_article/dev-snapshot-godot-4-3-dev-1.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+
+- Multiple controllers can now contribute to input actions ([GH-84943](https://github.com/godotengine/godot/pull/84943)), which expands upon previous fixes to multiple devices triggering input at the same time.
+
+- Profiler improvements for scripting ([GH-75623](https://github.com/godotengine/godot/pull/75623)) and rendering ([GH-85811](https://github.com/godotengine/godot/pull/85811)).
+
+- Node groups can now be configured project-wide ([GH-60965](https://github.com/godotengine/godot/pull/60965)), with scenes correctly updating when you make the changes.
+
+- Freed objects are now treated differently from null in comparison operators ([GH-73896](https://github.com/godotengine/godot/pull/73896)), which ensures consistency for programmers when managing instances and memory.
+
+- Loading of scene with corrupted or missing dependencies will no longer be aborted ([GH-85159](https://github.com/godotengine/godot/pull/85159)), allowing you to use and fix such scenes without external tools.
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-1.md]:
```suggestion
- Loading of scenes with corrupted or missing dependencies will no longer be aborted ([GH-85159](https://github.com/godotengine/godot/pull/85159)), allowing you to use and fix such scenes without external tools.
```
Or "Loading a scene..."

File: collections/_article/dev-snapshot-godot-4-3-dev-1.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+author: Yuri Sizov
+image: /storage/blog/covers/dev-snapshot-godot-4-3-dev-1.webp
+image_caption_title: "Yeetus"
+image_caption_description: "A game by YarnCat Games"
+date: 2023-12-21 17:00:00
+---
+
+This was an exciting year for Godot! Our contributors managed to deliver massive, feature-packed releases 3 times over our previous norm. We can't wait to see and share what the international community of passionate game and game engine developers can create for you in the year 2024. Actually, they have already started their work on the upcoming Godot 4.3 release.
+
+And it's a holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Please, enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with. It also contains a handful of fixes, including pretty much everything [published last week in Godot 4.2.1](/article/maintenance-release-godot-4-2-1/).
```
Comment (JohnVeness) [inline collections/_article/dev-snapshot-godot-4-3-dev-1.md]: The end sentence seems a little odd. Why would a 4.3 release *not* contain fixes released in 4.2.1? I mean, why mention it?

Comment (YuriSizov) [reply collections/_article/dev-snapshot-godot-4-3-dev-1.md]: We mention it so people can read those release notes and learn about things we don't mention in these release notes.

Edit: Also, incidentally, no, people don't always assume this is true that fixes from 4.2.1 are included in 4.3 🙂

Comment (JohnVeness) [reply collections/_article/dev-snapshot-godot-4-3-dev-1.md]: Ah OK. Maybe you could do something like:
```suggestion
And it's a holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Hot on the heals of [last week's release of Godot 4.2.1](/article/maintenance-release-godot-4-2-1/), please enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with, as well as some more fixes.
```

Comment (YuriSizov) [reply collections/_article/dev-snapshot-godot-4-3-dev-1.md]: I don't think this implies what I want it to imply. What feels wrong to you in the current structure of the statement?

Comment (mhilbrunner) [inline collections/_article/dev-snapshot-godot-4-3-dev-1.md]:
```suggestion
And it's holiday season, so we cannot leave you without a small gift as we wrap up the production and plan our leisure time. Please, enjoy the first dev snapshot of Godot 4.3, which includes quite a few noteworthy improvements for you to play with. It also contains a handful of fixes, including pretty much everything [published last week in Godot 4.2.1](/article/maintenance-release-godot-4-2-1/).
```

**Review by mhilbrunner — APPROVED** [review]
Tiniest nitpick I ever had. Don't feel strongly about it however. Great work!

File: collections/_article/dev-snapshot-godot-4-3-dev-1.md (hunk 2 above)
Comment (YuriSizov) [reply collections/_article/dev-snapshot-godot-4-3-dev-1.md]: Well, there may be other holiday seasons throughout the year :P But I'll take it.

Comment (JohnVeness) [reply collections/_article/dev-snapshot-godot-4-3-dev-1.md]: As I mentioned, it just felt odd to me saying 4.3 contains 4.2.1's fixes (although I note that you imply it's only most of them - have some been reverted, and if so, which?). But you know better than me what people might ask questions about, so if some people don't realise that 4.x release (that is released chronologically after 4.x-1.y) contains fixes from 4.x-1.y, then sure, keep the existing text!

Comment (YuriSizov) [reply collections/_article/dev-snapshot-godot-4-3-dev-1.md]: Yeah, we get asked this from time to time, and have to clarify. In this instance, it's just a device to add a link and avoid repeating the same high profile fixes already listed in the previous article.

As for me saying "pretty much", it's not really important. I can say that it includes everything. There are a couple of commits in 4.2.1 which were made specifically for 4.2 and aren't relevant/addressed differently in 4.3. But it shouldn't matter to most readers.

So if you think it's fine with these caveats, then I'd keep it as is 🙃 

PS. Thanks for your thorough reviews, by the way, always! They are very appreciated, and don't let me ever make you feel like they are not.

**Comment by YuriSizov** [comment]
Thanks for reviews!

**Pull request merged**